waku/v2/protocol/enr/localnode.go

```diff
@@ -120,9 +129,7 @@ func writeMultiaddressField(localnode *enode.LocalNode, addrAggr []multiaddr.Mul
 		fieldRaw = append(fieldRaw, maRaw...)
 	}
 
-	if len(fieldRaw) != 0 && len(fieldRaw) <= 100 { // Max length for multiaddr field before triggering the 300 bytes limit
```

Comment:
I am wondering if we need to have this check or some check for ENR length crossing 300 bytes, because i ran this locally with status-desktop and noticed my ENR after address update with circuit-relay to cross 300 bytes and no error was reported. 

`enr:-OO4QG48MPpNEqllrIT-BZ_KEELMSAyigEMAc279YNCkomB0GtsiXlwv2-dKAwZUV2qdR_zsxUKb_0WX3faYqMYzIuWGAZAz0qL2gmlkgnY0gmlwhMCoAQmKbXVsdGlhZGRyc7QAMgSnYxMvBnZfpQMnACUIAh`

<img width="1452" alt="Screenshot 2024-06-20 at 9 44 35 AM" src="https://github.com/waku-org/go-waku/assets/20137744/9f16af02-d28d-4bbf-aaa0-30c8dce04025">

Reply:
Looks like it was copy paste issue .. this is complete ENR which seems valid.

`enr=enr:-OO4QG48MPpNEqllrIT-BZ_KEELMSAyigEMAc279YNCkomB0GtsiXlwv2-dKAwZUV2qdR_zsxUKb_0WX3faYqMYzIuWGAZAz0qL2gmlkgnY0gmlwhMCoAQmKbXVsdGlhZGRyc7QAMgSnYxMvBnZfpQMnACUIAhIhAt60bRUEoHNuLlnsM12sU2PIQwBwfLIJ8a_ZPEY2-RnkgnJzhwAQAgBAACCJc2VjcDI1NmsxoQJ5lIphhEl668B0TOuIzNRyTUJCA_s4xvhAvQVqNa7CYYN0Y3CC2zKDdWRwguqLhXdha3UyDQ`

Reply:
Ah interesting! how did you generate this? When the ENR is signed, if the data exceeds 300 bytes, it should have generated an error as described in https://github.com/waku-org/go-waku/pull/1134#issuecomment-2179832821
Something like:
```
panic: enode: can't sign record: record bigger than 300 bytes [recovered]
	panic: enode: can't sign record: record bigger than 300 bytes

goroutine 6 [running]:
testing.tRunner.func1.2({0x9340e0, 0xc0000b8180})
	/usr/local/go/src/testing/testing.go:1545 +0x238
testing.tRunner.func1()
	/usr/local/go/src/testing/testing.go:1548 +0x397
panic({0x9340e0?, 0xc0000b8180?})
	/usr/local/go/src/runtime/panic.go:914 +0x21f
github.com/ethereum/go-ethereum/p2p/enode.(*LocalNode).sign(0xc0000f2300)
	/home/richard/go/pkg/mod/github.com/status-im/go-ethereum@v1.10.25-status.4/p2p/enode/localnode.go:310 +0x51f
github.com/ethereum/go-ethereum/p2p/enode.(*LocalNode).Node(0xc0000f2300)
	/home/richard/go/pkg/mod/github.com/status-im/go-ethereum@v1.10.25-status.4/p2p/enode/localnode.go:122 +0x172
```

I think this error is complaining that the ENR is incomplete, because ENRs use RLP encoding, and the first byte iirc indicates the data length. If the number of bytes is less than the data len, I'd imagine an error like this would be thrown.

Reply:
> Looks like it was copy paste issue .. this is complete ENR which seems valid.

you replied faster than my response :)